typings/index.d.ts

```diff
@@ -1,10 +1,45 @@
 import { AxiosInstance } from 'axios';
 
-interface RateLimitedAxiosInstance extends AxiosInstance {}
+export interface RateLimitedAxiosInstance extends AxiosInstance {
+   getMaxRPS(): number,
+   setMaxRPS(rps:number): void,
+   setRateLimitOptions(options: rateLimitOPtions): void,
+   // enable(axios: any): void,
+   // handleRequest(request:any):any,
+   // handleResponse(response: any): any,   
+   // push(requestHandler:any):any,
+   // shiftInitial():any,
+   // shift():any
+}
 
-declare function axiosRateLimit(
+type rateLimitOPtions = { maxRequests?: number, perMilliseconds?: number, maxRPS?:number };
```

Comment:
RateLimitOptions
typescript convention for type names is PascalCase

Reply:
should be "rateLimitOptions" not "rateLimitOPtions"

Reply:
@Jomel did you see my comment about PascalCase?
We recommend our contributors to follow these guidelines:
https://github.com/microsoft/TypeScript/wiki/Coding-guidelines
As you can see these are guidelines for contributors to the typescript repo, however, these are good guidelines for any typescript project.